content/docs/aro/blob-storage-csi/index.md

```diff
@@ -0,0 +1,315 @@
+
+    ```bash
+    export AAD_CLIENT_SECRET=REPLACE-WITH-YOUR-CLIENT-SECRET-VALUE
+    ```
+
+1. Set other environment variables:
+
+    > Update the `LOCATION`, `CLUSTER_NAME`, and `RG_NAME` variables to match your cluster details
+
+    ```bash
+    export LOCATION=eastus
+    export CLUSTER_NAME=my-cluster
+    export RG_NAME=myresourcegroup 
+    export TENANT_ID=$(az account show --query tenantId -o tsv)
+    export SUB_ID=$(az account show --query id)
+    export MANAGED_RG=$(az aro show -n $CLUSTER_NAME -g $RG_NAME --query 'clusterProfile.resourceGroupId' -o tsv)
+    export MANAGED_RG_NAME=`echo -e $MANAGED_RG | cut -d  "/" -f5`
+    ```
+
+1. Once we have all the environment variables set, we need to create the configuration file we will use to populate a json structure with all the data needed. 
+
+
+    ```bash
+    cat <<EOF >> cloud.conf
+    {
+    "tenantId": "$TENANT_ID",
+    "subscriptionId": $SUB_ID,
+    "resourceGroup": "$MANAGED_RG_NAME",
+    "location": "$LOCATION",
+    "useManagedIdentityExtension": false,
+    "aadClientId": "$AAD_CLIENT_ID",
+    "aadClientSecret": "$AAD_CLIENT_SECRET"
+    }
+    EOF
+    ```
+
+1. Check that all the attributes are populated. 
+
+>NOTE: Take care when executing this validation, since sensitive information should be disclosed in your screen. 
+
+   ```bash
+   cat cloud.conf
+   ```
+
+1. Create the project where you are going to install the driver.
+
+    ```bash
+    export CSI_BLOB_PROJECT=csi-azure-blob
+    oc new-project ${CSI_BLOB_PROJECT}
+    ```
+
+1. Then, create the secret into the project just created.
+
+    ```bash
+    export CSI_BLOB_SECRET=csi-azure-blob-secret
+    oc create secret generic ${CSI_BLOB_SECRET} --from-file=cloud-config=cloud.conf
+    ```
+
+
+## Driver installation
+
+Now, we need to install the driver, which could be done using a helm chart. 
+
+1. Assign permissions to the service accounts defined in the helm chart for the driver pods.
+
+    ```bash 
+    cat <<EOF | oc apply -f -
+    allowHostPorts: true
+    allowPrivilegedContainer: true
+    runAsUser:
+      type: RunAsAny
+    users:
+      - 'system:serviceaccount:${CSI_BLOB_PROJECT}:csi-blob-controller-sa'
+      - 'system:serviceaccount:${CSI_BLOB_PROJECT}:csi-blob-node-sa'
+    allowHostDirVolumePlugin: true
+    seccompProfiles: 
+      - '*'
+    seLinuxContext:
+      type: RunAsAny
+    metadata:
+      annotations:
+        kubernetes.io/description: >-
+          allows access to all privileged and host features and the
+          ability to run as any user, any group, any fsGroup, and with any SELinux
+          context.
+      name: csi-azureblob-scc
+    fsGroup:
+      type: RunAsAny
+    groups:
+      - 'system:cluster-admins'
+      - 'system:nodes'
+      - 'system:masters'
+    kind: SecurityContextConstraints
+    volumes:
+      - '*'
+    allowHostNetwork: true
+    apiVersion: security.openshift.io/v1
+    EOF
+
+    oc describe scc csi-azureblob-scc
```

Comment:
I don't think there's much value in describing the scc unless you want them to do something with the output.

Reply:
deleted.